src/Breadcrumb/Breadcrumb.scss

```diff
@@ -0,0 +1,2 @@
+@import "variables";
+@import "~bootstrap/scss/breadcrumb";
```

Comment:
Should this order be reversed? We would want our variables to override any boostrap variables of the same name, no?

Reply:
Variables in Paragon and bootstrap are declared with `!default` so first in gets priority. https://getbootstrap.com/docs/4.6/getting-started/theming/#variable-defaults

Reply:
Also, Bootstrap doesn't colocate their variables with component styles so this doesn't pull in the bootstrap ones anyway.